broadcast/src/linked/signer/actor.rs

```diff
@@ -36,8 +36,8 @@ use thiserror::Error;
 use tracing::{debug, error, info, warn};
 
 /// Represents a pending verification request to the application.
-struct Verify<C: Scheme, D: Array> {
-    start: SystemTime,
+struct Verify<C: Scheme, D: Array, E: Clock> {
+    _timer: histogram::Timer<E>,
```

Comment:
Why not use `PhantomData`?

Reply:
This was just a linting thing since `timer` is never used in the code unless I call `drop(timer)` later which is redundant (it automatically gets dropped after the `Verify` object is removed from the map once verification completes). So it wanted me to call it `_timer`. It is important that it is in the map though because I don't want it to get dropped until the verification returns...

Maybe the way to solve this is to explicitly call `drop(timer)` so that the linter doesn't complain